api/add_dimension.md

```diff
@@ -11,79 +11,62 @@ api:
-
-In a distributed hypertable, hash partitioning enables inserts to be
-parallelized across data nodes, even while the inserted rows share
-timestamps from the same time interval, and thus increases the ingest rate.
-Query performance also benefits by being able to parallelize queries
-across nodes, particularly when full or partial aggregations can be
-"pushed down" to data nodes (for example, as in the query
-`avg(temperature) FROM conditions GROUP BY hour, location`
-when using `location` as a hash partition). Please see our
-[best practices about partitioning in distributed hypertables][distributed-hypertable-partitioning-best-practices]
-for more information.
-
-### Parallelizing disk I/O on a single node
-
-Parallel I/O can benefit in two scenarios: (a) two or more concurrent
-queries should be able to read from different disks in parallel, or
-(b) a single query should be able to use query parallelization to read
-from multiple disks in parallel.
-
-Thus, users looking for parallel I/O have two options:
-
-1.  Use a RAID setup across multiple physical disks, and expose a
-single logical disk to the hypertable (that is, via a single tablespace).
-
-1.  For each physical disk, add a separate tablespace to the
-database. Timescale allows you to actually add multiple tablespaces
-to a *single* hypertable (although under the covers, a hypertable's
-chunks are spread across the tablespaces associated with that hypertable).
-
-We recommend a RAID setup when possible, as it supports both forms of
-parallelization described above (that is, separate queries to separate
-disks, single query to multiple disks in parallel).  The multiple
-tablespace approach only supports the former. With a RAID setup,
-*no spatial partitioning is required*.
-
-That said, when using hash partitions, we recommend using 1
-hash partition per disk.
-
-Timescale does *not* benefit from a very large number of hash
-partitions (such as the number of unique items you expect in partition
-field).  A very large number of such partitions leads both to poorer
+Every distinct item in hash partitioning is hashed to one of *N* buckets. By default, 
+TimescaleDB uses flexible range intervals to manage chunk sizes. The main purpose of hash
+partitioning is to enable parallelization across multiple disks within the same time 
+interval.
+
+### Parallelizing disk I/O 
+
+You use Parallel I/O in the following scenarios:
+
+- Two or more concurrent queries should be able to read from different disks in parallel.
+- A single query should be able to use query parallelization to read from multiple disks in parallel.
+
+For the following options:
+
+- **RAID**: use a RAID setup across multiple physical disks, and expose a single logical disk to the hypertable.
+  That is, using a single tablespace.
+
+  Best practice is to use RAID when possible. This is because it supports the concurrent and single
+  scenarios and *no spatial partitioning is required*.
```

Comment:
This was probably there before and I am not sure where it comes from, but this seems to imply that using multiple tablespaces/disks requires spatial partitioning. This is not the case (and I don't think it has ever been the case). You can spread chunks across multiple disks with only time partitioning, so I am not sure exactly why "no spatial partitioning is required" is mentioned since it applies also the "Multiple tablespaces".

I would recommend just removing this last bit.

If we want to mention that RAID is "best practice" (and, honestly, I am not exactly sure why it is best practice), I would motivate it primarily with the fact that you need not manually manage tablespaces in the DB when using RAID. Still, you do need the complexity of a RAID setup instead.